std/typecons.d

```diff
@@ -3275,6 +3288,50 @@ auto nullable(T)(T t)
 
     assert(foo.approxEqual(bar));
 }
+// bugzilla issue 19037
+@safe unittest
+{
+    import std.datetime : SysTime;
+
+    struct Test
+    {
+        bool b;
+        @disable this();
+        this(bool b) { this.b = b; }
+        nothrow invariant { assert(b == true); }
```

Comment:
Ehm a bit of spacing wouldn't hurt here.

Reply:
fixed